Makefile

```diff
@@ -134,21 +139,21 @@ reboot:
 	$(ECHO_V)rm -rf ./e2etest/*chain*.db
 	$(GOBUILD) -o ./bin/$(BUILD_TARGET_SERVER) -v ./$(BUILD_TARGET_SERVER)
 	export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH):$(PWD)/crypto/lib:$(PWD)/crypto/lib/blslib
-	./bin/$(BUILD_TARGET_SERVER) -config-path=e2etest/config_local_delegate.yaml
+	./bin/$(BUILD_TARGET_SERVER) -config-path=e2etest/config_local_delegate.yaml $(SourceFlags)
```

Comment:
It doesn't sound reasonable that the version info is not burned into the executable, but the arguments at run time